### PR TITLE
Add support for MachO/ARM64

### PR DESCRIPTION
**Pull request by derekparker**

### CODE REVIEW

File: testdata/gold
Comment (TcM1911) [inline testdata/gold:1]: Can you remove these from the PR please?

Comment (derekparker) [reply testdata/gold:1]: Done.

**Comment by TcM1911** [comment]
Do you have anymore fixes or should I cut a new release?

**Comment by derekparker** [comment]
> Do you have anymore fixes or should I cut a new release?

You can go ahead and cut a new release, this is the last patch I had planned.

**Pull request merged**